codegen/arrai/svc_types.arrai

```diff
@@ -105,6 +105,36 @@ let orderedTypes = \types
                         >> $`${go.name(.@item('name').s)} ${go.type(.@item('type'))}`
                     ::\i}
                 }
+            ${
```

Comment:
This looks like a duplicate to the json validation already merged.

Reply:
ya that would be resolved during merge.

Reply:
This is a pull request to merge to master. If this file shouldn't be changed from what master is presently then this file wouldn't show any changes.